go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	rsc.io/letsencrypt v0.0.3 // indirect
 	sigs.k8s.io/controller-runtime v0.6.3
 	sigs.k8s.io/controller-tools v0.3.0
-	sigs.k8s.io/kubebuilder v1.0.9-0.20201014221150-c993a2a221fe
+	sigs.k8s.io/kubebuilder v1.0.9-0.20201021204649-36124ae2e027
```

Comment:
Is it the latest version?

Reply:
It's not. This is the commit that added the fix for this particular issue after my upstream PR merged. I didn't want to pull in any more than necessary since this PR is meant to only cover the rbac/v1 scaffolding fix.